config/software/datadog-gohai.rb

```diff
@@ -17,14 +20,11 @@
 build do
   ship_license "https://raw.githubusercontent.com/DataDog/gohai/#{version}/LICENSE"
   ship_license "https://raw.githubusercontent.com/DataDog/gohai/#{version}/THIRD_PARTY_LICENSES.md"
-  # Go get gohai
-  command "#{gobin} get -d -u github.com/DataDog/gohai", :env => env
-  # Checkout gohai's deps
-  command "#{gobin} get -u github.com/shirou/gopsutil", :env => env
-  command "git checkout v2.0.0", :env => env, :cwd => "#{Omnibus::Config.cache_dir}/src/datadog-gohai/src/github.com/shirou/gopsutil"
-  command "#{gobin} get -u github.com/cihub/seelog", :env => env
-  command "git checkout v2.6", :env => env, :cwd => "#{Omnibus::Config.cache_dir}/src/datadog-gohai/src/github.com/cihub/seelog"
+  # dep manager
+  command "#{gobin} get github.com/Masterminds/glide", env: env
   # Checkout and build gohai
-  command "git checkout #{version} && git pull", :env => env, :cwd => "#{Omnibus::Config.cache_dir}/src/datadog-gohai/src/github.com/DataDog/gohai"
-  command "cd #{env['GOPATH']}/src/github.com/DataDog/gohai && #{gobin} run make.go #{gobin} && mv gohai #{install_dir}/bin/gohai", :env => env
+  command "git fetch && git checkout #{version} && git pull", env: env
```

Comment:
I don't think we need to fetch/checkout/pull gohai anymore now that you've defined a github source